src/visitors/transpileCssProp.js

```diff
@@ -19,14 +19,10 @@ export default t => (path, state) => {
   if (!state.required) {
     if (!bindings.styled) {
       state.items.push(
-        t.variableDeclaration('var', [
-          t.variableDeclarator(
-            t.identifier('styled'),
-            t.callExpression(t.identifier('require'), [
-              t.stringLiteral('styled-components'),
-            ])
-          ),
-        ])
+        t.importDeclaration(
```

Comment:
Looks good. Though maybe just doing `const styled = require('styled-components').default` will be better than adding an import since it seems to add a lot of extra code.

Reply:
I think going with imports make more sense, since some people might not even transpile imports potentially?